src/scss/layout/_footer.scss

```diff
@@ -68,6 +68,7 @@ footer {
 			a {
 				color: $white;
 				font-weight: $font-weight-normal;
+				white-space: nowrap;
```

Comment:
Although #381 is not an issue and a fix is not needed, just for your future reference, this solution of using `white-space: nowrap` is not ideal.

If you turn on these UIO settings with the "Galaxy Fold" screen size:

1. set "text style" to "Open Dyslexic";
2. Turn on "Enhance inputs";
3. Turn on "Syllables".

This `nowrap` change will force the email stretch out of the viewport and cause the horizontal overflow.

Your change seems working for a specific scenario but it actually introduces a more serious problem when experimenting with more scenarios. Even with the specific scenario you described, this change is too data driven that only works with the current length of an email address. What if the email is changed to a longer email, or what if a smaller screen appears in the market?

Whenever possible, it's preferred that designs and technical solutions are able to accommodate future changes. In this case, a better solution is to allow the email/text to display in multiple lines rather than forcing it to fit into one line.

The problem with this solution is a generic problem with all solutions you issued with WeCount project: you are too focus on fixing an issue with the current data on the page rather than thinking broadly of how your solutions could work in general regardless of data changes and setting changes etc. 

Please stop filing issues and pull requests before you understand what proper solutions are meant to be. Thanks.